data_warehouse/services/coverage_statistics_service.py

```diff
@@ -88,6 +89,19 @@ def groupby_titles(users_qs):
                         technical_title=item['technical_title']).count()
                 }
             )
+        # merge titles which we are not interested in into single group
+        # labled with other_label
+        if interest_titles is not None:
+            item_other = {
+                'title': other_label, 'coverage_count': 0, 'total_count': 0
+                }
+            for item in data:
+                if item['title'] not in interest_titles:
+                    item_other['coverage_count'] += item['coverage_count']
+                    item_other['total_count'] += item['total_count']
+                    data.remove(item)
```

Comment:
I think this would cause a bug if you remove item from a list during iteration. Suggestion:
```python
removed_titles = set()
for item in data:
    if item['title'] not in interest_titles:
        item_other['coverage_count'] += item['coverage_count']
        item_other['total_count'] += item['total_count']
        removed_titles.add(item['title'])
data = [x for x in data if x['title'] not in removed_titles]
```